disk_password_gui/disk_password_gui.py

```diff
@@ -148,8 +160,10 @@ def perform_setup(self):
         # pylint: disable=attribute-defined-outside-init
         self.builder = Gtk.Builder()
 
-        self.builder.add_from_file(pkg_resources.resource_filename(
-            'disk_password_gui', 'disk_password_gui.glade'))
+        glade_ref = importlib.resources.files(
+            __name__).joinpath('disk_password_gui.glade')
```

Comment:
This fails for me with `TypeError: 'disk_password_gui.disk_password_gui' is not a package`